src/components/forms/components/LinkToDropdown.js

```diff
@@ -1,7 +1,8 @@
 'use strict';
 
 import React from 'react';
-import { DropdownButton, DropdownItem } from 'react-bootstrap';
+import DropdownButton from 'react-bootstrap/esm/DropdownButton';
+import DropdownItem from 'react-bootstrap/esm/DropdownItem';
```

Comment:
Can't you do `import { DropdownButton, DropdownItem } from 'react-bootstrap/esm/';`?